config/org.yaml

```
@@ -649,6 +658,14 @@ orgs:
         privacy: closed
         repos:
           pdig: maintain
+      peribolos-syncer-maintainers:
+        description: maintainers of falcosecurity/perperibolos-syncer
+        maintainers:
+          - maxgio92
```

```suggestion
          - maxgio92
          - leogr
```

If you like, count me in! :smiley_cat: